conf/test.config

```
@@ -46,26 +46,29 @@ params {
 }
 
 process {
-    withName:'.*:FREEC_SOMATIC'{
-        ext.args = {
-            [
-                "sample":[
-                    inputformat: "pileup",
-                    mateorientation: "FR"
-                ],
-                "general":[
-                    bedgraphoutput: "TRUE",
-                    noisydata: "TRUE",
-                    minexpectedgc: "0",
-                    readcountthreshold: "1",
-                    sex: meta.sex,
-                    window: "10",
+
+    if (params.tools && params.tools.split(',').contains('controlfreec')) {
+        withName:'.*:FREEC_SOMATIC'{
```

```suggestion
        withName:'FREEC_SOMATIC'{
```

this should be enough, no?

---

yeah probably, unsure why we had this 🤔 testing...